frontend/src/js/controllers/authCtrl.js

```diff
@@ -7,11 +7,13 @@
         .module('evalai')
         .controller('AuthCtrl', AuthCtrl);
 
-    AuthCtrl.$inject = ['utilities', '$state', '$rootScope', '$timeout'];
+    AuthCtrl.$inject = ['utilities', '$state', '$rootScope', 'configService'];
 
-    function AuthCtrl(utilities, $state, $rootScope) {
-        var vm = this;
+    function AuthCtrl(utilities, $state, $rootScope,configService) {
 
+        var vm = this;
+        console.log(configService);
```

Comment:
Make sure that there should be no console when you remove `WIP` from title of this PR